add more quantized activation benchmarks and input sizes

vkuzo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35731 qactivation_benchmarks: small bug fix
* **#35729 add more quantized activation benchmarks and input sizes**
* #35389 add benchmark for quantized batchnorm

Summary:

* there were a few quantized activations which had implementations but not benchmarks, adds them
* adds the input sizes from `unary_tests.py` here, so we can compare fairly from fp to quantized implementations of activations

Test Plan:

```
python -m pt.qactivation_test
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D20875544](https://our.internmc.facebook.com/intern/diff/D20875544)

vkuzo committed:
add more quantized activation benchmarks and input sizes

Summary:

* there were a few quantized activations which had implementations but not benchmarks, adds them
* adds the input sizes from `unary_tests.py` here, so we can compare fairly from fp to quantized implementations of activations

Test Plan:

```
python -m pt.qactivation_test
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

z-a-f reviewed (approved):
lgtm

z-a-f commented on benchmarks/operator_benchmark/pt/qactivation_test.py:
```diff
@@ -32,8 +32,13 @@
+        (3, 4, 5),      # Rank=3
+        (2, 3, 4, 5),    # Rank=4,
+        # Dimensions from the floating point benchmarks
+        (512, 512),
+        (256, 1024),
+    ),
```
Be careful with adding large benchmarks in the short_configs -- those are run every time, and if they are too slow, the test will take forever.

vkuzo replied:
makes sense.  The times to execute these are still reasonable so should be ok, compared to some more expensive short tags checked in (like fp batchnorm).

vkuzo committed:
Update on "add more quantized activation benchmarks and input sizes"


Summary:

* there were a few quantized activations which had implementations but not benchmarks, adds them
* adds the input sizes from `unary_tests.py` here, so we can compare fairly from fp to quantized implementations of activations

Test Plan:

```
python -m pt.qactivation_test
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]
